docs/source/index.rst

```diff
@@ -0,0 +1,28 @@
+.. Service Kit documentation master file, created by
+   sphinx-quickstart on Fri Feb 21 13:58:09 2025.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+.. include:: ../../README.md
+   :parser: myst_parser.sphinx_
+   :start-line: 0
+   :end-line: 40
+
+
+.. include:: ../../README.md
+   :parser: myst_parser.sphinx_
+   :start-line: 116
+   :end-line: 145
```

Comment:
Doing this by specifying lines is going to be problematic. Whenever we update the file these lines need to be updated too.